### DISCUSSION TITLE
Using process.env in renovate.json doesn't work

### DISCUSSION THREAD

**Discussion by lmilbaum** [renovatebot/renovate#22695] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

I have few gitlab ci/cd variables which I'd like to be consumed by renovate.json

### What you tried so far.

I've tried using  process.env.<VAR NAME> as a placeholder for the value with no avail

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
JSON files can't ever access process.env. You would instead need to use something like `config.js` so that it can
